kube-flagd-proxy/cmd/start.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/open-feature/flagd/core/pkg/logger"
+	"github.com/open-feature/flagd/core/pkg/service"
+	syncServer "github.com/open-feature/flagd/core/pkg/service/sync"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap/zapcore"
+)
+
+// start
+
+const (
+	logFormatFlagName   = "log-format"
+	metricsPortFlagName = "metrics-port"
+	portFlagName        = "port"
+)
+
+func init() {
+	flags := startCmd.Flags()
+
+	// allows environment variables to use _ instead of -
+	flags.Int32P(portFlagName, "p", 8013, "Port to listen on")
+	flags.Int32P(metricsPortFlagName, "m", 8014, "Metrics port to listen on")
```

Comment:
Should the proxy use different ports?

Reply:
Good point. Yes, this makes sense for example when running locally or for e2e tests

Reply:
updated